buildifier/utils/tempfile.go

```diff
@@ -25,8 +26,10 @@ func (tf *TempFile) WriteTemp(data []byte) (file string, err error) {
 	return name, nil
 }
 
+// Clean removes all created temporary files
```

Comment:
should this func be Close() error ? 

Since os.Remove returns error (that should be handled but is not currently)?

Reply:
Done